### DISCUSSION TITLE
error dalam Exporting plots with ggpubr

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#562] [Q&A] [answered]
codenya sudah benar, namun masih tetap error, kira2 ini salah dimana ya? ketika semua plotnya sudah jadi dan ingin dibuat gabung jadi 3 plot berbaris ini 

<img width="1440" alt="Screen Shot 2023-02-22 at 13 50 33" src="https://user-images.githubusercontent.com/123171973/220546082-67d22014-1d8e-4a1c-9e7b-164160592426.png">

**Comment by divakartika** [comment, marked as answer]
Dear @mosesnugraha ,

Error ini masih berkaitan dengan Issue #555 , silakan merujuk ke issue tersebut :)

Best regards,
Team Algoritma

**Reply by mosesnugraha** [reply]
halo m4a @divakartika terkait issue #555 sudah terselesaikan, ternyata salah codenya. Mengenai case #562 ini saya bingung letak salahnya dimana? bisa dijelaskan lebih detail lagi ga mba? karena yang #555 Sudah bisa dan benar hasiilnya, tapi sekarang ini yang #562 yang bener2 bingung2

**Reply by divakartika** [reply]
Dari error message nya, error tersebut disebabkan oleh hal yang sama, yaitu tidak terdapat kolom `complaint` dalam dataframe. Jika dilakukan run all chunk above apakah masih error? Dapat dilampirkan pula file .Rmd nya untuk kami telaah lebih lanjut (dapat diupload ke Google Drive / Github dan dituliskan link nya di sini)
